drivers/storage/isilon/storage/isilon_storage.go

```diff
@@ -174,27 +174,20 @@ func (d *driver) getVolumeAttachments(ctx types.Context) (
 		return nil, err
 	}
 
-	// should be using this, but instanceID is coming back either blank or
-	// with metadata today
-	// iid := ctx.InstanceID()
-
-	ii, err := d.InstanceInspect(ctx, nil)
-	if err != nil {
-		return nil, err
-	}
-	iid := ii.InstanceID
-
-	ld, err := d.LocalDevices(ctx, nil)
-	if err != nil {
-		return nil, err
+	iid, iidOK := ctx.InstanceID()
+	if iidOK {
+		ld, err := d.LocalDevices(ctx, nil)
```

Comment:
Hi @Oskoss,

I believe in Slack I also pointed out that one should not be calling `d.LocalDevices`. Also, you must have an old build, because `ctx.InstanceID()` is invalid. It is `context.InstanceID(ctx)`. Additionally, it would be `context.LocalDevices(ctx)`.